frontend/indigenousapprenticeship/src/js/components/Dialog/ApplyDialog.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import Dialog from "@common/components/Dialog";
+
+import type { BasicDialogProps } from "./types";
+
+const ApplyDialog: React.FC<BasicDialogProps> = ({ isOpen, onDismiss }) => {
+  const intl = useIntl();
+  return (
+    <Dialog
+      isOpen={isOpen}
+      onDismiss={onDismiss}
+      color="ia-secondary"
+      title={intl.formatMessage({
+        defaultMessage: "Apply Now",
+        description: "Heading for the apply now dialog",
+      })}
+    >
+      <p>
+        {intl.formatMessage(
+          {
+            defaultMessage:
+              "Please send your resume and cover letter explaining your passion for IT and why you're interested in joining the program to: <mailLink>edsc.pda-iap.esdc@hrsdc-rhdcc.gc.ca</mailLink>. A team member will contact you in 3-5 business days",
+            description: "First paragraph for apply now dialog",
+          },
+          {
+            // TO DO: We should fix this, im just unsure how to atm
```

Comment:
I think the idea is to move the function about the return function so that you don't get a new instance of the function every time.
![image](https://user-images.githubusercontent.com/8978655/166253835-bbb2533c-d035-4db5-a645-42c1edb2ef2b.png)

Reply:
Thanks, that makes sense.

Resolved with [move mail link component outside of dialog](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2539/commits/c2b47309448b049f6c554cf0a602f6a761610e8a)